pkg/features/features.go

```diff
@@ -43,6 +50,7 @@ func init() {
 // defaultKubernetesFeatureGates consists of all known feature keys specific to external-provisioner.
 // To add a new feature, define a key for it above and add it here.
 var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	Topology:             {Default: false, PreRelease: featuregate.GA},
-	HonorPVReclaimPolicy: {Default: false, PreRelease: featuregate.Alpha},
+	Topology:                         {Default: false, PreRelease: featuregate.GA},
+	HonorPVReclaimPolicy:             {Default: false, PreRelease: featuregate.Alpha},
+	CrossNamespaceSourceProvisioning: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
we also need to add this feature gate to the readme doc

Reply:
Could you tell me where should i write it?

Reply:
Added CrossNamespaceVolumeDataSource in [README.md](https://github.com/kubernetes-csi/external-provisioner/pull/805/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5).